pkg/provider/branch/provider.go

```diff
@@ -270,27 +267,23 @@ func (b *branchENIProvider) UpdateResourceCapacity(instance ec2.EC2Instance) err
 
 // ReconcileNode reconciles a nodes by getting the list of pods from K8s and comparing the result
 // with the internal cache.
-func (b *branchENIProvider) ReconcileNode(nodeName string) (ctrl.Result, error) {
+func (b *branchENIProvider) ReconcileNode(nodeName string) bool {
 	trunkENI, isPresent := b.getTrunkFromCache(nodeName)
 	log := b.log.WithValues("node", nodeName)
 	if !isPresent {
-		log.Info("stopping the reconcile job")
-		return ctrl.Result{}, nil
+		log.Info("no trunk ENI is pointing to the given node", "NodeName", nodeName)
+		return true
 	}
 	podList, err := b.apiWrapper.PodAPI.ListPods(nodeName)
 	if err != nil {
 		log.Error(err, "failed fo list pod")
-		return reconcileRequeueRequest, nil
-	}
-	err = trunkENI.Reconcile(podList.Items)
-	if err != nil {
-		b.log.Error(err, "failed to reconcile")
-		return reconcileRequeueRequest, nil
+		return true
```

Comment:
Same question as above.

Reply:
Returning `true` for these two uncommon errors is making sure we keep the shortest interval for next clean up since we don't know what exactly happen and want to retry again asap.

Reply:
Got it, can we add a comment here?

Reply:
Updated comments, thanks